src/twitterWidget.directive.js

```diff
@@ -12,16 +12,17 @@ function TwitterWidget(ngTweetLogger, TwitterWidgetFactory) {
         transclude: true,
         scope: {
             twitterWidgetId: '=',
-            twitterWidgetOptions: '='
+            twitterWidgetOptions: '@'
```

Comment:
Since you're accessing this attribute via `attrs`, it isn't necessary to declare it in this block.